README.md

```diff
@@ -43,53 +43,132 @@ exports.passport = {
 ### Using github and twitter strategy
 
 ```js
-// app.js
+// config/config.default.js
+exports.passport = {
+  github: {
```

Comment:
so the build-in support for `egg-passport` is using  `config.passport.github`, and other such as `egg-passport-dingtalk` / `egg-passport-whatever` which is not maintenance by us, should use `config.passportDingtalk` ?